test/inductor/test_torchinductor.py

```diff
@@ -718,6 +719,74 @@ def fn(a, b):
             ),
         )
 
+    def test_torch_compile_override_registration(self):
+        dynamic = True
+        namespace_name = "aten"
+        dispatch_key = "CPU"
```

Comment:
Can we test CUDA as well?

Reply:
Sure. I will refine the test case a little bit.